packages/core/addon/components/navi-visualization-config/apex-pie.js

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright 2020, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * Usage:
+ * <NaviVisualizationConfig::ApexPie
+ *   @request=request
+ *   @response=response
+ *   @options=tableOptions
+ * />
+ */
+import Component from '@glimmer/component';
+
+export default class NaviVisualizationConfigApexPieComponent extends Component {}
```

Comment:
Do we need this file? Can we make this template only?

Reply:
The next PR will have this chart's config panel, which uses this file, but I can remove it from this PR if you want

Reply:
either way